projects/documentation/content/guides/generating-components.md

```diff
@@ -13,50 +13,44 @@ The fastest way to get started with creating a new component is to generate it f
 $ yarn new-package
 
 ? SWC package name (i.e. color-area)
-? Spectrum CSS package name (i.e. colorarea)
 ```
 
-In response to the prompt above, the package name should be the kebab case version of the Spectrum CSS pattern that you are looking to add to the repo. That is, the `SpectrumPattern` that you want to create should be represented as `spectrum-pattern` in your response. From here, the command generates a new package for your pattern with the following file structure in the `/packages` directory:
-
 ```
-> spectrum-pattern
+> package-name
    > src/
        index.ts
-       SpectrumPattern.ts
-       spectrum-pattern.css
+       SpectrumPackageName.ts
+       package-name.css
+       package-name-overrides.css
+       spectrum-package-name.css
+       spectrum-package-name-overrides.css
        spectrum-config.js
    > test/
        > benchmark/
            basic-test.ts
-       spectrum-pattern.test.ts
+       package-name.test.ts
    > stories/
-       spectrum-pattern.stories.ts
-   sp-spectrum-pattern.ts
+       package-name.stories.ts
+   sp-package-name.ts
    README.md
    tsconfig.json
    package.json
 ```
 
-From here, peek into the `package.json` file and ensure the "devDependency" of `@spectrum-css/spectrumpattern` (replacing "spectrumpattern" with the pattern you're implementing, such as "badge" or "tooltip") listed there is of the most current version.
-
 Outside of your new package, you will need to manually add information about your new package to the `tsconfig-all.json` and `tsconfig-react-wrapper.json` files, as well as the `bundle` package:
 
-Open `tsconfig-all.json`, find "references", and add an entry for your package (`{ "path": "packages/spectrum-pattern" }`) alphabetically (replacing "spectrum-pattern" with the pattern you're implementing, such as "action-button" or "progress-bar"). The `tsconfig-all.json` config is used to build types for the project in parallel with the JS build that is handled outside of `tsc`. This will ensure that the types for your new package are available throughout the library, include at demonstration and test time.
+Open `tsconfig-all.json`, find "references", and add an entry for your package (`{ "path": "packages/package-name" }`) alphabetically (replacing "package-name" with the pattern you're implementing, such as "action-button" or "progress-bar"). The `tsconfig-all.json` config is used to build types for the project in parallel with the JS build that is handled outside of `tsc`. This will ensure that the types for your new package are available throughout the library, include at demonstration and test time.
```

Comment:
Just some grammar nits:

```suggestion
Open `tsconfig-all.json`, find "references", and add an entry for your package (`{ "path": "packages/package-name" }`), alphabetically (replacing "package-name" with the pattern you're implementing, such as "action-button" or "progress-bar"). The `tsconfig-all.json` config is used to build types for the project in parallel with the JS build that is handled outside of `tsc`. This will ensure that the types for your new package are available throughout the library, including at demonstration and test time.
```